rllib/core/models/catalog.py

```diff
@@ -98,22 +98,33 @@ def __init__(
         self.model_config_dict = {**MODEL_DEFAULTS, **model_config_dict}
         self.view_requirements = view_requirements
 
-        # Produce a basic encoder config.
+        self.latent_dims = None
+        # Overwrite this post-init hook in subclasses
+        self.__post_init__()
+
+    def __post_init__(self):
```

Comment:
This __post_init__ can be easily overwritten (rather than __init__) to get rid of the encoder config.
Another thing Jun wanted.